src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PackageReferenceCommands/ListPackage/ListPackageCommand.cs

```diff
@@ -157,12 +179,52 @@ private static ReportType GetReportType(bool isDeprecated, bool isOutdated, bool
             throw new ArgumentException(string.Format(Strings.ListPkg_InvalidOptions));
         }
 
+        private static (IReportRenderer, ReportOutputFormat) GetOutputType(string outputFormatOption, string outputVersionOption)
+        {
+            ReportOutputFormat outputFormat = ReportOutputFormat.Console;
+            if (!string.IsNullOrEmpty(outputFormatOption))
+            {
+                try
+                {
+                    outputFormat = EnumExtensions.GetValueFromName<ReportOutputFormat>(outputFormatOption);
```

Comment:
why not use enum parse?

Reply:
![image](https://user-images.githubusercontent.com/8766776/182512124-1cf675ec-bdd1-42a5-9371-c54a5e4940f8.png)
As you can see enum parsing is case sensitive. Another way of dealing with this one is make upper case the `outputFormatOption` and also create enum all Upper case too.